src/controller/c/messages.h

```diff
@@ -115,6 +115,8 @@
 #define C_SUPERVISOR_VIRTUAL_REALITY_HEADSET_IS_USED 98
 #define C_SUPERVISOR_VIRTUAL_REALITY_HEADSET_GET_POSITION 99
 #define C_SUPERVISOR_VIRTUAL_REALITY_HEADSET_GET_ORIENTATION 100
+#define C_SUPERVISOR_FIELD_GET_COUNT 101
```

Comment:
This message name could be misunderstood I would rename it to `C_SUPERVISOR_NODE_GET_FIELD_COUNT`.